primeros_pasos.txt

```diff
@@ -0,0 +1,13 @@
+
+
+
+
+Comentarios a tener en cuanta, para mi persona.
```

Comment:
please use English, as much as possible

Reply:
I forgot not to upload that document sry. I'll just remove it